test/runner.py

```diff
@@ -2,6 +2,9 @@
 import os
 import sys
 
+if __name__ == '__main__':
+    sys.path.insert(0, os.path.dirname(os.path.dirname(__file__)))
```

Comment:
don't relative imports imply only looking in the dir where the current module is located?

not really sure here, trying to find alternatives, the hack just feels wrong (but I guess it's not a blocker)